app/containers/Settings.js

```diff
@@ -106,113 +57,228 @@ class Settings extends Component
 
         const voipDevices = [];
         let selectedDevice = 0;
-
         for (let i = 0; i < this.props.voip.devices.length; ++i)
         {
             let device = this.props.voip.devices[i];
             voipDevices.push({ value: device.id, label: device.name });
 
-            if (device.id === this.props.voip.selectedDevice)
+            if (device.id === this.props.voip.selectedDevice) {
                 selectedDevice = i;
+            }
         }
 
         if (voipDevices.length === 0)
         {
             voipDevices.push({ value: -1, label: 'No microphone detected' });
-            selectedDevice = 0;
+            selectedDevice = 0; // I don't think we need this one
         }
 
-        let gameSettings = (
+        let audioSettings = (

```

Comment:
Might be a good idea to move these into their own components (audio, game, mods).